demo/capture_config_ci.yaml

```diff
@@ -0,0 +1,100 @@
+lcr-modules:
```

Comment:
I think there is no Snakefile that uses this config

Reply:
I've modified the structure of the CI tests to be self-contained in their own folder (ci), so its easier to understand and follow